src/app.ts

```diff
@@ -0,0 +1,16 @@
+import { ApolloServer, AuthenticationError } from "apollo-server";
+import { typeDefs } from "./schema";
+import { resolvers } from "./resolvers";
+import { logger } from "./logger";
+import { authenticationContextMiddleware } from "./middleware/authentication";
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+  context: authenticationContextMiddleware,
+});
+
+server.listen().then(({ url }) => {
+  logger.info(`Server started at ${url}`);
+  console.log(`Server started at ${url}`);
```

Comment:
nit: You don't need the `console.log` here anymore.

Reply:
Absolutely right, I removed it.